pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/TimeAverageMessageData.java

```diff
@@ -177,4 +175,40 @@ public double totalMsgRate() {
     public double totalMsgThroughput() {
         return msgThroughputIn + msgThroughputOut;
     }
+
+    @Override
+    public int compareTo(TimeAverageMessageData other) {
+        int result = this.compareByBandwidthIn(other);
+
+        if (result == 0) {
+            result = this.compareByBandwidthOut(other);
+        }
```

Comment:
Why do we have separate bandwithIn and Out checks here? What if some bundles' bandwithOut is way greater than bandwithIn?

Reply:
I imitate the logic implemented by 
https://github.com/apache/pulsar/blob/master/pulsar-client-admin-api/src/main/java/org/apache/pulsar/policies/data/loadbalancer/NamespaceBundleStats.java#L40-L47

Reply:
To me, this seems like another bug. but I agree that we are not fixing this issue in this PR>